workflow.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "U.S. Greenhouse Gas Center: Data Workflow Documentation"
+---
+
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data workflow documentation. View data flow diagrams to discover the journey of each dataset from acquisition into the US GHG Center.
+
+Data flow diagrams are grouped topically and labeled by dataset name. The dataset product type (model output, satellite observation, etc.) is noted next to each. Click on a dataset name to view the data flow diagram for that dataset, which summarizes the process followed to bring the dataset into the US GHG Center.
```

Comment:
"The dataset product type (model output, satellite observation, etc.) is noted next to each." - we can probably remove this